spec/spec_helper_acceptance.rb

```diff
@@ -14,4 +14,27 @@
   end
 end
 
+RSpec.configure do |c|
```

Comment:
This is not what I meant. I've already tried telling you in https://github.com/theforeman/puppet-pulpcore/pull/347#pullrequestreview-2157476682 that you need to move https://github.com/theforeman/puppet-pulpcore/blob/master/spec/acceptance/hieradata/os/RedHat/8.yaml because version 3 of `voxpupuli-acceptance` ends up using a different hierarchy. Rather than overriding the hierarchy to the old style, you should move the file.

In practice what you need to do is exactly the reverse of https://github.com/theforeman/puppet-pulpcore/commit/c16ddb8acd3c3b7209c122b4e5807cb0e22dafb3.

Reply:
@ekohl I am not sure what I was missing out but isn't my this change https://github.com/archanaserver/puppet-pulpcore/commit/9f884fb511707c7721503bf9ebc03eb2890b1d21 was exactly reverting https://github.com/theforeman/puppet-pulpcore/pull/299/commits/4248c03610f38a30133daf9390a9fc2a52ba90e0 one?

Also I'm not sure if I did the right thing now, but i did revert this specific https://github.com/theforeman/puppet-pulpcore/commit/c16ddb8acd3c3b7209c122b4e5807cb0e22dafb3 and pushed it and I can see this now "Skipping EOL operating system CentOS 8"